### PR TITLE
Add maximum size warning for NextMap blocks

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
Relates to https://github.com/up42/blocks/pull/626

### CODE REVIEW

**Comment by rodrigoalmeida94** [comment]
@perusio if you can have a look ASAP that would be great, we're planning on merging today.

File: source/up42-blocks/data/nextmap-5m.rst [hunk 1]
```diff
@@ -27,6 +27,10 @@ For more information on supported filters, see :ref:`query filter section  <filt
 * ``intersects``: A GeoJSON geometry to use as an AOI. Will return all data points that are within this geometry. Use only ``bbox``
   **or** ``intersects``.
 
+.. warning::
+  This block can return a maximum of 1000 tiles per job. This means that the extent
+  of your input geometry can have a **maximum area of 5000 sqkm**.
```
Comment (markusuwe) [inline source/up42-blocks/data/nextmap-5m.rst:32]: If the tile size is fixed at 1000x1000 does that not mean that one of these tiles has 25 SQKM and therefore the overall max size would be 25,000 SQKM?

File: source/up42-blocks/data/nextmap-10m.rst [hunk 2]
```diff
@@ -27,6 +27,10 @@ For more information on supported filters, see :ref:`query filter section  <filt
 * ``intersects``: A GeoJSON geometry to use as an AOI. Will return all data points that are within this geometry. Use only ``bbox``
   **or** ``intersects``.
 
+.. warning::
+  This block can return a maximum of 1000 tiles per job. This means that the extent
+  of your input geometry can have a **maximum area of 10 000 sqkm**.
```
Comment (markusuwe) [inline source/up42-blocks/data/nextmap-10m.rst:32]: If the tile size is fixed at 1000x1000 does that not mean that one of these tiles has 100 SQKM and therefore the overall max size would be 100,000 SQKM?

File: source/up42-blocks/data/nextmap-5m.rst (hunk 1 above)
Comment (rodrigoalmeida94) [reply source/up42-blocks/data/nextmap-5m.rst]: For sure, my maths are really bad ahahaa

**Pull request merged**